legal-api/src/legal_api/models/alias.py

```diff
@@ -48,7 +48,7 @@ def json(self):
         """Return a dict of this object, with keys in JSON format."""
         alias = {
             'id': self.id,
-            'alias': self.alias,
+            'name': self.alias,
```

Comment:
edit-ui need a change for this. For alteration in edit-ui this response has been used.

Reply:
Ya Thats right. Edit-UI will need an update. alias is a property in the model. In the json it should be in sync with what we defined in schema